app/helper/ethereum/helper.go

```diff
@@ -114,6 +114,19 @@ func GetAddressBySignature(hash []byte, signature []byte) (string, error) {
 	return crypto.PubkeyToAddress(*pubKey).String(), nil
 }
 
+func RecoverSignerFromStr(signature string, authMsgBytes []byte) (string, string, error) {
```

Comment:
Can we add names to return values for clarification what is the first and second value